Cinema-Reservation-System-Front-End-Client-Part-Internship-project/src/app/authentication/login/login.component.html

```diff
@@ -0,0 +1,43 @@
+<div class="register flex flex-col items-center justify-center">
+  <h1 class="font-bold">Login</h1>
+  <form class="flex flex-col" [formGroup]="loginForm" (ngSubmit)="onSubmit()">
+    <div>
+      <app-input-field
+        label="Email"
+        type="email"
+        formControlName="email"
+      ></app-input-field>
+      @if (
+        email?.invalid && (email?.dirty || email?.touched || formSubmitted)
+      ) {
+        @if (email?.hasError("required")) {
+          <p>Email is required</p>
+        }
+        @if (email?.hasError("email")) {
+          <p>Email is invalid</p>
+        }
+      }
+    </div>
+
+    <div>
+      <app-input-field
+        label="Password"
+        type="password"
+        formControlName="password"
+      ></app-input-field>
+      @if (
+        password?.invalid &&
+        (password?.dirty || password?.touched || formSubmitted)
+      ) {
+        @if (password?.hasError("required")) {
+          <p>Password is required</p>
+        }
+        @if (password?.hasError("minlength")) {
+          <p>Password must be at least 5 characters long</p>
+        }
+      }
+    </div>
+
+    <app-button label="Login"></app-button>
```

Comment:
Lets disable the button if the form has errors or if the form is currently submitting (the request to the server is sent, but we haven't got the response yet)

Reply:
Also, add an @Input to the buttton component with 'type' property, so we can set it to the 'submit' type